src/checkbox/src/Checkbox.js

```diff
@@ -104,13 +109,15 @@ class Checkbox extends PureComponent {
   static defaultProps = {
     checked: false,
     indeterminate: false,
+    innerRef: () => {},
     onChange: () => {},
     appearance: 'default'
   }
 
-  setIndeterminate = el => {
+  handleInnerRef = el => {
     if (!el) return
     el.indeterminate = this.props.indeterminate
+    this.props.innerRef(el)
```

Comment:
wouldn't you want to bubble the `innerRef` even when `!el`?

```suggestion
    if (el) {
      el.indeterminate = this.props.indeterminate
    }
    this.props.innerRef(el)
```

Reply:
Yeah, that makes sense.